ReSharper.FSharp/test/src/FSharp.Tests/NavigationTests.fs

```diff
@@ -28,7 +28,7 @@ type FSharpGoToUsagesTest() =
 
     [<Test>] member x.``Record Ctor 01 - Source``() = x.DoNamedTest()
 
-    [<TestReferences("FSharpRecord.dll")>]
+    [<TestReferences("../../../assemblies/FSharpRecord.dll")>]
```

Comment:
Could you extract the literal to a place like the `Common.fs` file?
```fsharp
[<RequireQualifiedAccess>]
module TestAssemblies =
    let [<Literal>] FSharpRecord = "../../../assemblies/FSharpRecord.dll"
```

Reply:
Not sure how to make it work with relative paths properly. Will update it later.